content/ui-patterns/degraded-experiences.mdx

```diff
@@ -0,0 +1,535 @@
+      width="456"
+    />
+    <Caption>Render an error message in place of the content.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/design/assets/2313998/6a0cc580-3a0a-4023-9e34-e6387e0629b5"
+      alt="Image of Memex table with 'undefined of undefined'"
+      width="456"
+    />
+    <Caption>Don't attempt to render UI that is missing critical information.</Caption>
+  </Dont>
+</DoDontContainer>
+
+#### Panels and larger areas
+
+<img
+  width="960"
+  alt="Two images: 1. comment box; 2. blankslate in place of comment box"
+  src="https://github.com/primer/design/assets/2313998/8321cdb9-8bc2-4f7c-8107-ff535d12498f"
+/>
+
+If the affected area is large enough, replace the affected UI with a [blankstate](/components/blankslate) component that explains why the expected UI isn't there.
+
+#### Dialogs (modal and non-modal)
+
+If the content of a dialog is not critical and cannot be rendered, prevent the dialog from even being opened. For non-critical dialogs that appear on hover, remove the hover interaction. For non-critical dialogs that appear on click, remove the button that triggers the dialog. For example, if a user's profile data cannot is unavailable, don't show a hovercard when their avatar is hovered.
+
+If the dialog is a core part of a workflow, replace the content of the dialog with a message explaining why the expected UI isn't there. If you're using a dialog component that supports error states (for example, [select panel](/components/select-menu#select-panel)), follow the component's guidelines for rendering error messages.
+
+<img
+  width="960"
+  alt="Two images: 1. Label or assignee SelectPanel 2. A degraded versin of that"
+  src="https://github.com/primer/design/assets/2313998/653cc04b-e88e-4235-8c04-47b8dba57351"
+/>
+
+If you're not using a component that supports error states, replace the content of the dialog with a [blankstate](/components/blankslate) component explaining why the expected UI isn't there.
+
+<img
+  width="960"
+  alt="Two images: 1. 'Edit details' modal that appears when you edit a column in a Memex project board 2. A degraded versin of that"
+  src="https://github.com/primer/design/assets/2313998/bde4367e-8881-41f5-9d06-a2ea5cd79fd6"
+/>
+
+## Degraded navigation
+
+Some links can be static (for example, `https://github.com/`), but many links are made from dynamic data (for example: `https://github.com/{org}/{repo}/issues/{issue-number}`).
+
+When a dynamic link in the navigation is not yet available, fall back to not rendering it.
+
+If the page is in a state where we're not sure if a link is available, the navigation item should be put in a loading state. Refer to the pattern guidelines for [loading](/ui-patterns/loading) for more information.
+
+### Global navigation header
+
+<img
+  width="960"
+  alt="Example of a degraded global nav"
+  src="https://github.com/primer/design/assets/2313998/3427b08a-e30d-43b0-80b3-2f3431a90461"
+/>
```

Comment:
Should the purple labels on these images say 
`organization link cannot be loaded` instead of `org link cannot be`?